uqcsbot/scripts/crates.py

```diff
@@ -48,41 +50,44 @@ class SlackBlock(ABC):
 
     @abstractmethod
     def get_formatted_block(self) -> Dict[str, str]:
-        """Gets the dictionary which maps the required properties for the given block"""
+        """
+        Gets the dictionary which maps the required properties for the given block
+        """
         pass
 
 
 class ImageBlock(SlackBlock):
-    """SlackBlock that represents an image block (contains an image url and alternate text for that image)"""
+    """
+    SlackBlock that represents an image block
+    (contains an image url and alternate text for that image)
+    """
 
     def __init__(self, url: str, alt_text: str):
         self.url = url
         self.alt_text = alt_text
 
     def get_formatted_block(self):
-        return {
-            'type': 'image',
-            'image_url': self.url,
-            'alt_text': self.alt_text
-        }
+        return {'type': 'image', 'image_url': self.url, 'alt_text': self.alt_text}
 
 
 class TextBlock(SlackBlock):
-    """SlackBlock that represents a text block (contains an image url and alternate text for that image)"""
+    """
+    SlackBlock that represents a text block
+    (contains an image url and alternate text for that image)
+    """
 
     def __init__(self, text: str, markdown: bool = True):
         self.text = text
         self.markdown = markdown
 
     def get_formatted_block(self):
-        return {
-            'type': 'mrkdwn' if self.markdown else 'plain_text',
-            'text': self.text
-        }
+        return {'type': 'mrkdwn' if self.markdown else 'plain_text', 'text': self.text}
```

Comment:
Combining these lines onto the one line obfuscates the meaning. Either split the lines or calculate the value for 'type' prior.

Reply:
Made more readable.